### PR TITLE
Add GolangCI lint

### PR DESCRIPTION
**Pull request by tochka**

### CODE REVIEW

**Review by Scratch-net — APPROVED** [review]

File: utils/errcheck.go [hunk 1]
```diff
@@ -49,90 +49,105 @@ var (
...
-		return "", ErrIncorrectAppToken
+	return result
+}
+
+func isAuthFailed(err *client.VirgilAPIError) bool {
+	result := err.Code == 40019 || err.Code == 40027
+	if len(err.Errors) != 0 {
+		errField := err.Errors[0]
+
+		result = result || errField.Code == 40001 && strings.Contains(errField.Message, "Invalid email or password")
```
Comment (Scratch-net) [inline utils/errcheck.go:125]: why check for message if you have the code?

File: cmd/app/list.go [hunk 2]
```diff
@@ -38,30 +38,31 @@ package app
...
-
 			var apps []*models.Application
 			apps, err = listFunc(vcli)
 
 			if err != nil {
 				return err
 			}
 
-			defaultApp, err := utils.LoadDefaultApp()
+			defaultApp, _ := utils.LoadDefaultApp()
```
Comment (Scratch-net) [inline cmd/app/list.go:65]: Why ignore error?

**Review by Scratch-net — CHANGES_REQUESTED** [review]

File: utils/errcheck.go (hunk 1 above)
Comment (tochka) [reply utils/errcheck.go:125]: That code was before. I can suppose that we use the same code but different messages.

File: cmd/app/list.go (hunk 2 above)
Comment (tochka) [reply cmd/app/list.go:65]: the err variable was unused. You can see dafaultApp variable use for setup defaultAppID variable and it is used for mark in application lists an application as default. 
https://github.com/VirgilSecurity/virgil-cli/blob/golangci/cmd/app/list.go#L86

**Review by Scratch-net — APPROVED** [review]

**Review by hogaco — APPROVED** [review]

**Pull request merged**